src/screens/Analytics/Logs/LogUtils/LogUtils.res

```diff
@@ -5,3 +5,117 @@ let sortByCreatedAt = (log1: JSON.t, log2: JSON.t) => {
   let keyB = log2->getKey
   compareLogic(keyA, keyB)
 }
+
+type flowType =
+  | PaymentsCancel
+  | PaymentsCapture
+  | PaymentsConfirm
+  | PaymentsCreate
+  | PaymentsStart
+  | PaymentsUpdate
+  | RefundsCreate
+  | RefundsUpdate
+  | DisputesEvidenceSubmit
+  | AttachDisputeEvidence
+  | RetrieveDisputeEvidence
+  | IncomingWebhookReceive
+  | NotDefined
+
+let itemToObjMapper = flowString => {
```

Comment:
name can be more specific of what it's doing

Reply:
this is the std name we use for string to object converter

Reply:
can be named as **flowTypeStringToTypeMapper**